
# [CVE-2019-20432](http://lustre.org/)

## Description

- `In the Lustre file system before 2.12.3, the mdt module has an out-of-bounds access and panic due to the lack of validation for specific fields of packets sent by a client. mdt_file_secctx_unpack does not validate the value of name_size derived from req_capsule_get_size.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 7.8

## Scores

- **Exploitability Score**:
  - 10.0
- **Impact Score**:
  - 6.9
- **Base Severity**:
  - HIGH

## Other Information

- **Publish Date**:
  - 2020-01-27 05:15:12
- **Vulnerability Status**:
  - Analyzed

## References

- **cve@mitre.org**: http://lustre.org/
- **cve@mitre.org**: http://wiki.lustre.org/Lustre_2.12.3_Changelog
- **cve@mitre.org**: https://jira.whamcloud.com/browse/LU-12604
- **cve@mitre.org**: https://review.whamcloud.com/#/c/35868/
